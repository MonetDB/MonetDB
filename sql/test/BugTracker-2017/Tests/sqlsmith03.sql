-- And a new run after 234K queries
-- !42S02!SELECT: no such table 'querylog_calls'


select
  8 as c0
from
  tmp.keys as ref_68
      left join sys.auths as ref_70
        inner join sys._columns as ref_86
            inner join tmp.keys as ref_101
                inner join sys.querylog_calls as ref_102
                on (ref_101.type = ref_102.cpu )
              inner join sys.idxs as ref_103
              on (ref_102.io = ref_103.id )
            on (ref_86.table_id = ref_102.cpu )
          right join sys.privileges as ref_111
              left join sys.systemfunctions as ref_112
                left join sys.querylog_calls as ref_113
                on (ref_112.function_id = ref_113.cpu )
              on (ref_111.privileges = ref_112.function_id )
            right join sys.schemas as ref_114
              inner join sys.geometry_columns as ref_115
                right join sys.users as ref_116
                on (ref_115.f_table_name = ref_116.name )
              on (ref_114.owner = ref_115.srid )
            on (ref_111.obj_id = ref_116.default_schema )
          on (ref_103.id = ref_113.cpu )
        on (ref_70.id = ref_111.obj_id )
      on (ref_68.rkey = ref_114.id )
    left join sys.columns as ref_121
        inner join sys.types as ref_122
        on (ref_121.type_digits = ref_122.id )
      inner join tmp._tables as ref_123
      on (ref_122.schema_id = ref_123.id )
    on (ref_116.default_schema = ref_123.id )
where cast(coalesce(cast(coalesce(ref_101.table_id,
      ref_121.number) as int),
    ref_68.table_id) as int) is NULL
limit 164; 
