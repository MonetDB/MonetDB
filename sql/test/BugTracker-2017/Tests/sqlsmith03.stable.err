stderr of test 'sqlsmith03` in directory 'sql/test/BugTracker-2017` itself:


# 17:13:50 >  
# 17:13:50 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=33167" "--set" "mapi_usock=/var/tmp/mtest-17366/.s.monetdb.33167" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/niels/scratch/monetdb/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2017"
# 17:13:50 >  

# builtin opt 	gdk_dbpath = /home/niels/scratch/monetdb/Linux-x86_64/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 33167
# cmdline opt 	mapi_usock = /var/tmp/mtest-17366/.s.monetdb.33167
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbpath = /home/niels/scratch/monetdb/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2017
# cmdline opt 	gdk_debug = 536870922

# 17:13:51 >  
# 17:13:51 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-17366" "--port=33167"
# 17:13:51 >  


# 17:13:51 >  
# 17:13:51 >  "Done."
# 17:13:51 >  

