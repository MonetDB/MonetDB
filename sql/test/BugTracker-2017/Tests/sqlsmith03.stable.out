stdout of test 'sqlsmith03` in directory 'sql/test/BugTracker-2017` itself:


# 23:03:00 >  
# 23:03:00 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=30090" "--set" "mapi_usock=/var/tmp/mtest-6840/.s.monetdb.30090" "--set" "monet_prompt=" "--forcemito" "--dbpath=/export/scratch1/mk/default//Linux/var/MonetDB/mTests_sql_test_BugTracker-2017"
# 23:03:00 >  

# MonetDB 5 server v11.26.0
# This is an unreleased version
# Serving database 'mTests_sql_test_BugTracker-2017', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 15.589 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2017 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://vienna.da.cwi.nl:30090/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-6840/.s.monetdb.30090
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded


# 23:03:00 >  
# 23:03:00 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-6840" "--port=30090"
# 23:03:00 >  

#select
#  8 as c0
#from
#  tmp.keys as ref_68
#      left join sys.auths as ref_70
#        inner join sys._columns as ref_86
#            inner join tmp.keys as ref_101
#                inner join sys.querylog_calls as ref_102
#                on (ref_101.type = ref_102.cpu )
#              inner join sys.idxs as ref_103
#              on (ref_102.io = ref_103.id )
#            on (ref_86.table_id = ref_102.cpu )
#          right join sys.privileges as ref_111
#              left join sys.systemfunctions as ref_112
#                left join sys.querylog_calls as ref_113
% . # table_name
% c0 # name
% tinyint # type
% 1 # length

# 23:03:00 >  
# 23:03:00 >  "Done."
# 23:03:00 >  

